pkg/cmd/roachtest/tests/validate_system_schema_after_version_upgrade.go

```diff
@@ -134,7 +134,13 @@ func validateSystemSchemaAfterUpgradeTest(
 	mvt.Run()
 
 	// Start a cluster with the latest binary and get the system schema
-	// from the cluster.
+	// from the cluster. Mark all processes as expected to die before wiping.
+	t.Monitor().ExpectProcessDead(c.All())
```

Comment:
`c.Wipe` already calls `c.t.Monitor().ExpectProcessDead(selectedNodesOrDefault(nodes, c.All()))` so this should be unnecessary. No harm in calling it again but more curious if you ran into any issues without this line as that would indicate a bug.